File: 1_6/1_6.rb
def check_char(str)
    glas_let = "aeiouyAEIOUY"
    soglas_let = "bcdfghjklmnpqrstvwxyzBCDFGHJKLMNPQRSTVWXYZ"
    glas_count = 0
    soglas_count = 0
    
    str.each_char do |ch| / (сhar ch : str) - аналог / 
        if glas_let.include?(ch)
            glas_count += 1
        elsif soglas_let.include?(ch)
            soglas_count += 1
        end
    end
    
    if glas_count > soglas_count
        return "Да"
    elsif glas_count < soglas_count
        return "Нет"
    else
        return "Одинаково"
    end
end

puts "Enter your string:"
str = gets.chomp
puts check_char(str)

File: 1_6/1_6.js
function checkChar(str){ // сравниваем кол во гласных и согласных букв
    const glasLet = "aeiouyAEIOUY";
    const soglasLet = "bcdfghjklmnpqrstvwxyzBCDFGHJKLMNPQRSTVWXYZ";
    let glasCount = 0;
    let soglasCount = 0;


    for (let i = 0; i < str.length; i++) {
        if (glasLet.includes(str[i])) {
            glasCount++;
        } else if (soglasLet.includes(str[i])) {
            soglasCount++;
        }
    }


    if (glasCount > soglasCount) {
        return "Да";
    } else if (glasCount < soglasCount) {
        return "Нет";
    } else if (glasCount == soglasCount ){
        return "Одинаково";
    } 
}


console.log(checkChar(prompt("Enter your string "))); // promt - выводит своё сообщенеи и вводит строку